extensions/amp-sidebar/validator-amp-sidebar.protoascii

```diff
@@ -93,5 +93,6 @@ tags: {  # amp-sidebar > nav
   child_tags: {
     mandatory_num_child_tags: 1
     child_tag_name_oneof: "UL"
+    child_tag_name_oneof: "OL"
```

Comment:
There is some text in `amp-sidebar.md` that also needs to be updated, namely: "The nav containing the toolbar attribute must only contain a single `<ul>`"